frontend/components/ClubEditPage/ApplicationsCard.tsx

```diff
@@ -103,10 +111,13 @@ const ApplicationModal = (props: {
             {questionType !== null &&
               questionType === ApplicationQuestionType.FreeResponse && (
                 <Field
+                  validate={validateWordCount}
                   name="word_limit"
                   as={TextField}
                   type={'number'}
-                  helpText={'Word limit for this free response question'}
+                  helpText={
+                    'Word limit for this free response question (max 500)'
+                  }
```

Comment:
I see what you mean, definitely misunderstood the ask. I'll make it so that when applications are being designed, the total wc / commitee is enforced to be <= 500, so that we can have defined word limits per question (so that students aren't stuck in some weird optimisation problem where they're trying to stay under the limit across all their responses).

Or alternatively, do you think this would be a good newbie task? Probably not super crucial to implement.

Reply:
Potentially would work as a newbie project but this change probably needs to be implemented by the beginning of the Wharton Council recruiting cycle which would be before we've onboarded newbies I think